config/tests/jobs/jobs_test.go

```diff
@@ -1086,25 +1086,96 @@ func TestSigReleaseMasterBlockingOrInformingJobsShouldUseFastBuilds(t *testing.T
 	}
 }
 
+// extractUsesCIBucket returns true if kubetest --extract=foo
```

Comment:
Is this also true for kubetest2?

Reply:
I couldn't tell that `--extract=` was pulling in anything weird so didn't feel the need to add further "is this definitely invoking kubetest in a way we recognize" checks.  

/cc @amwat 
I think enforcing for kubetest2 is going to require some changes to kubetest2 and the tests here, so I'm opting to ignore for followup later.
- I don't think `kubetest2-gce` has support for extracting pre-built versions of kubernetes yet, its readme states it ned
- `kubetest2-tester-ginkgo` has `--test-package-bucket (default "kubernetes-release")` as a start

There may also be other subprojects that pull from `kubernetes-release-dev` via scripts we can't introspect here, which I'd like to defer to the broader "remove gs://kubernetes-release-dev" effort which might need to be KEP'ed.

Reply:
yeah we don't do published released versions for GCE yet in kubetest2